### PR TITLE
Change redirectURL to Support Build web on Github Deployment

### PR DESCRIPTION
**Pull request by hoangdat**
https://github.com/linagora/twake-on-matrix/assets/6462404/b379ffd5-4924-4c2b-9a09-fee749c920f1

Screenshot 2023-08-29 at 06 05 02

### CODE REVIEW

**Comment by hoangdat** [comment]
@tk-nguyen please check, but need to make sure to find exactly `auth.html` in your deployment to complete OIDC

File: lib/pages/connect/connect_page_mixin.dart [hunk 1]
```diff
@@ -50,14 +50,13 @@ mixin ConnectPageMixin {
...
-    final redirectUrl = kIsWeb
-        ? '${html.window.origin!}/web/auth.html'
-        : '${AppConfig.appOpenUrlScheme.toLowerCase()}://login';
+    final redirectUrl = _generateRedirectUrl();
     final url = _getAuthenticateUrl(
       context: context,
       id: id,
       redirectUrl: redirectUrl,
     );
+    print('datph::::::: $url');
```
Comment (sherlockvn) [inline lib/pages/connect/connect_page_mixin.dart:59]: remove

File: config.sample.json [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
   "application_welcome_message": "Welcome to Twake!",
   "default_homeserver": "matrix.linagora.com",
-  "web_base_url": "https://twakematrix-on-commit.twake.app/web",
   "privacy_url": "https://twake.app/en/privacy/",
-  "render_html": false,
+  "render_html": true,
   "hide_redacted_events": false,
-  "hide_unknown_events": false
+  "hide_unknown_events": false,
+  "issue_id": 231
```
Comment (tk-nguyen) [inline config.sample.json:9]: Should this be default to `null`?

**Review by sherlockvn — APPROVED** [review]

**Comment by hoangdat** [comment]
@tk-nguyen , merged

**Pull request merged**